index.html

```diff
@@ -57,6 +57,10 @@ <h2>contact</h2>
           <a href="https://github.com/esrefbozkurt" target="_blank">GitHub</a>
         </section>
       </main>
+<<<<<<< HEAD
+    </div>
+=======
     </section>
+>>>>>>> main
   </body>
```

Comment:
Here you still have some remaining stuff from the merge.
Try to close the HTML tags properly (`</section></body>`?).